geometry_msgs/PointStamped explore_center
geometry_msgs/PolygonStamped explore_boundary
---
---
geometry_msgs/PoseStamped next_frontier
geometry_msgs/PoseStamped base_position